01_variables/more_user_input_of_data.py

```diff
@@ -1,14 +1,34 @@
-# The user is asked for several pieces of information. Display it on the screen as a summary. 
+# The user is asked for several pieces of information. Display it on the screen as a summary. 
 # First name: Helena
-Last name: Bonham-Carter
-Grade (9-12): 12
-Student ID: 453916
-Login: bonham_453916
-GPA (0.0-4.0): 3.73
-Information-
-Login:bonham_453916 
-ID:453916 
-Name:Bonham-Carter; Helena 
-GPA:3.73 
-Grade:12 
-# for more info on this quiz, go to this url: http://www.programmr.com/more-user-input-data-3
\ No newline at end of file
+# Last name: Bonham-Carter
+# Grade (9-12): 12
+# student ID: 453916
+# Login: bonham_453916
+# GPA (0.0-4.0): 3.73
+# Information-
+# Login:bonham_453916
+# ID:453916
+# Name:Bonham-Carter; Helena
+# GPA:3.73
+# Grade:12
+# for more info on this quiz, go to this url: http://www.programmr.com/more-user-input-data-3
+
+
+def profile():
+    first_name = input("Enter your first name:")

```

Comment:
I would prefer to have a space between the question and the answer the user will give.

```python
first_name = input("Enter your first name: ")
```
which will render as follows
```shell
Enter your first name: 45
```
Notice the space between user input and the question. It helps in cleaning up and making the program look goo to the user, in web app development, we call this UI/UX